test/cql/cdc_delta_modes_test.result

```diff
@@ -24,7 +24,7 @@ select "cdc$batch_seq_no", "cdc$operation", pk, ck, i1 from tb2_scylla_cdc_log w
 			"pk" : "1"
 		},
 		{
-			"cdc$batch_seq_no" : "1",
+			"cdc$batch_seq_no" : "2",
```

Comment:
Hmm, this change in the test indicates the previous code had a bug, which you fixed now?

Reply:
Nope. That shows that there is a bug in my code. We are inc:ing seq:no one time to many. Woops.

Reply:
fixed.